lib/mix/lib/mix/tasks/profile.cprof.ex

```diff
@@ -0,0 +1,328 @@
+    * `--require`, `-r` - requires pattern before running the command
+    * `--parallel`, `-p` - makes all requires parallel
+    * `--no-compile` - does not compile even if files require compilation
+    * `--no-deps-check` - does not check dependencies
+    * `--no-archives-check` - does not check archives
+    * `--no-halt` - does not halt the system after running the command
+    * `--no-start` - does not start applications after compilation
+    * `--no-elixir-version-check` - does not check the Elixir version from mix.exs
+
+  ## Profile output
+  
+  Example output:
+                                                                           CNT
+      Total                                                                 15
+      Enum                                                                   6  <--
+        Enum."-map/2-lists^map/1-0-"/2                                       4
+        Enum.reverse/1                                                       1
+        Enum.map/2                                                           1
+      :elixir_compiler                                                       4  <--
+        anonymous fn/1 in :elixir_compiler.__FILE__/1                        3
+        anonymous fn/0 in :elixir_compiler.__FILE__/1                        1
+      String.Chars.Integer                                                   3  <--
+        String.Chars.Integer.to_string/1                                     3
+      :erlang                                                                2  <--
+        :erlang.trace_pattern/3                                              2
+      Profile done over 20229 matching functions
+
+  The default output contains data gathered from all matching functions. The left
+  column structures each module and its total call count trace is presented on the right.
+  Each module has its count discriminated by function below. The `<--` symbol is meant to
+  help visualize where a new module call count begins.
+
+  The first row (Total) is the sum of all function calls. In the last row the number of
+  matching functions that were considered for profiling is presented.
+
+  When `--from-mfa` option is specified, call count tracing will be started only for 
+  the functions matching the given MFA:
+
+      String.Chars.Integer                                                   3  <--
+        String.Chars.Integer.to_string/1                                     3
+      Profile done over 1 matching functions
+
+  In the example above the MFA `"{String.Chars.Integer, :to_string, :_}"`
+  was given, therefore only one function was matched. The MFA must be a tuple
+  with `module`, `function` and `arity` (by this order). However, notice that `function` and 
+  `arity` may be arbitrary by passing the `:_` atom. Erlang modules have to be passed as 
+  atoms, similarly to calling Erlang modules in code (e.g., `:ets`, `:inet`, `:math`). Elixir
```

Comment:
We don't use `e.g.` and `i.e.` abbreviations, as they are confusing, especially for non-native speakers. Please use "for example".